Documentation - Consolidation of documentation around building new/custom connectors + consistency on vocabulary (#868)
airbytehq/airbyte: docs/changelog.md

# Changelog

> If you're interested in our changelog on which new connectors are available out of the box on Airbyte, please check our [connector changelog](integrations/integrations-changelog.md).

Reviewer: ```suggestion
This is the changelog for Airbyte core. For our connector changelog, please visit our [Connector Changelog](integrations/integrations-changelog.md) page.
```
